OWNERS

```diff
@@ -1,17 +1,25 @@
 approvers:
   - dtrifiro
+  - fialhocoelho
   - heyselbi
-  - rpancham
+  - joerunde
```

Comment:
@z103cb raises a good point, Nick Hill ([njhill](https://github.com/njhill)) is missing

Reply:
👍 yeah how did we forget him

Reply:
@heyselbi would you mind adding Nick so we can merge this?